pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -406,6 +406,12 @@ type Interface struct {
 	LinkState           *LinkState       `xml:"link,omitempty"`
 	FilterRef           *FilterRef       `xml:"filterref,omitempty"`
 	Alias               *Alias           `xml:"alias,omitempty"`
+	DHCPOptions         []DHCPOption     `xml:"dhcpOptions,omitempty"`
```

Comment:
Do we not have access to the vmi definition where we start the dhcp server?

Reply:
I don't think we do or at least I didn't see it thats why I used the xml schema as kind of a holding area a bit weird I know. If you know of better ways would love to change it.

Reply:
I think it makes sense to extend `StartDHCP` at https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/network/common.go#L61 to also pass in extra options, or pass in the VMI. Something along that line.